content-services/latest/install/ansible.md

```diff
@@ -2,7 +2,7 @@
 title: Install with Ansible
 ---
 
-This page describes how to install Content Services using an [Ansible](https://www.ansible.com){:target="_blank"} playbook. Ansible is an open-source software provisioning, configuration management, and application installation tool that enables infrastructure as code. Alfresco provides an Ansible playbook capable of installing Content Services (Enterprise Edition) version 7.3.
+This page describes how to install Content Services using an [Ansible](https://www.ansible.com){:target="_blank"} playbook. Ansible is an open-source software provisioning, configuration management, and application installation tool that enables infrastructure as code. Alfresco provides an Ansible playbook capable of installing Content Services (Enterprise Edition) version .

```

Comment:
It's either missign the version number or the word "version" should be removed completely? I'd rather we remove any mention of version given the playbook can install several versions.

Reply:
I agree - removing the version number also reduces the maintenance we have to do for each new release.

Reply:
I removed the need to update each version by adding your suggestion Alex.